mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.c

```diff
@@ -212,23 +187,6 @@ static void dcesrv_NspiUnbind(struct dcesrv_call_state *dce_call,
 		DCESRV_NSP_RETURN(r, MAPI_E_LOGON_FAILED, NULL);
 	}
 
-	/* Step 1. Retrieve handle and free if emsabp context and session are available */
```

Comment:
Still think that in an explicit disconnect from a client we should release the session (this is an operation not an unbind from rpc layer)